notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-756_analysis.ipynb

```diff
@@ -0,0 +1,701 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #23.        fig.savefig(r"/home/s/seanmacb/SITCOM/SITCOM-756_local/{}_actuator_m1m3.jpg".format(start))
</pre><p>I suggest to create a "plot" sub-directory within the notebook directory to store the figures</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/135/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>